src/actions/__tests__/collapseContext.ts

```diff
@@ -292,3 +293,125 @@ describe('context view', () => {
       - z`)
   })
 })
+
+describe('collapsing contexts with special attributes', () => {
+  it('should delete =pin when collapsing a context', () => {
+    const steps = [
+      importText({
+        text: `
+          - a
+            - b
+              - =pin
+                - true
+              - c
+            - d
+          - e
+        `,
+      }),
+      toggleHiddenThoughts,
```

Comment:
You can omit `toggleHiddenThoughts` from reducer tests. That was just for visibility when manually testing and shouldn't affect the result of the `collapseContext` command.